eBay-brand-tracker/ebay_bt_venv/Lib/site-packages/_distutils_hack/__init__.py

```
@@ -0,0 +1,239 @@
+        "using distutils directly, ensure that setuptools is installed in the "
+        "traditional way (e.g. not an editable install), and/or make sure "
+        "that setuptools is always imported before distutils."
+    )
+
+
+def clear_distutils():
+    if 'distutils' not in sys.modules:
+        return
+    import warnings
+
+    warnings.warn(
```

Ruff

No explicit `stacklevel` keyword argument found

(B028)